Makefile

```diff
@@ -247,7 +247,8 @@ arch_freebsd_amd64_host:
 	env HOSTOS=freebsd HOSTARCH=amd64 $(MAKE) host
 
 arch_freebsd_amd64_target:
-	env TARGETOS=freebsd TARGETARCH=amd64 $(MAKE) target
+	env GOOG=freebsd GOARCH=amd64 $(GO) install ./syz-fuzzer
+	env TARGETOS=freebsd TARGETARCH=amd64 $(MAKE) fuzzer execprog stress
```

Comment:
But why?
Building freebsd executor works on linux today, and with your patches it should also work on freebsd. So why are we excluding it?

Reply:
The changes at executor/executor_bsd.h make it impossible to build freebsd executor on linux.

Reply:
I think we need to revert changes in executor/executor_bsd.h to not lose valuable test coverage.
On linux we open coded ioctl constants too (nobody has up-to-date headers installed in system), works just fine.